.github/workflows/pylint.yml

```diff
@@ -0,0 +1,23 @@
+name: Pylint
```

Comment:
Can you please update the description of this PR and what is the purpose for this pylint ? Thank you.